dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/worker/task/sql/SqlTask.java

```diff
@@ -355,9 +355,8 @@ private PreparedStatement prepareStatementAndBind(Connection connection, SqlBind
         }
         Map<Integer, Property> params = sqlBinds.getParamsMap();
         if(params != null){
-            for(Integer key : params.keySet()){
-                Property prop = params.get(key);
-                ParameterUtils.setInParameter(key,stmt,prop.getType(),prop.getValue());
+            for(Map.Entry<Integer, Property> entry: sqlBinds.getParamsMap().entrySet()) {
+                ParameterUtils.setInParameter(entry.getKey(),stmt,entry.getValue().getType(),entry.getValue().getValue());
```

Comment:
To make it easier to read, Here, I would sugguest to change to:

Property prop = entry.getValue();
ParameterUtils.setInParameter(entry.getKey(), stmt, prop.getType(), prop.getValue());

Reply:
ok, i will modify it, thanks.